process/block/shardblock.go

```diff
@@ -1912,6 +1912,12 @@ func (sp *shardProcessor) createMiniBlocks(haveTime func() bool, randomness []by
 	if sp.flagScheduledMiniBlocks.IsSet() {
 		miniBlocks = sp.scheduledTxsExecutionHandler.GetScheduledMiniBlocks()
 		sp.txCoordinator.AddTxsFromMiniBlocks(miniBlocks)
+
+		scheduledTxs := sp.scheduledTxsExecutionHandler.GetScheduledIntermediateTxs()
```

Comment:
a unit test for this will ensure the code will not get broken by accident

Reply:
done